src/main/java/javax/validation/Configuration.java

```diff
@@ -121,6 +122,20 @@
 	T parameterNameProvider(ParameterNameProvider parameterNameProvider);
 
 	/**
+	 * Defines the clock provider. Has priority over the configuration
```

Comment:
See the javadoc for `traversableResolver`. I supposed it refers to the one from the validation.xml conf file.